plugins/modules/ovirt_template.py

```diff
@@ -845,6 +873,49 @@ def _get_vnic_profile_mappings(module):
     return vnicProfileMappings
 
 
+def import_template(module, connection):
+    templates_service = connection.system_service().templates_service()
+    if search_by_name(templates_service, module.params['name']) is not None:
+        return False
+
+    events_service = connection.system_service().events_service()
+    last_event = events_service.list(max=1)[0]
+
+    external_template = module.params['kvm']
+    imports_service = connection.system_service().external_template_imports_service()
+    imported_template = imports_service.add(
+        otypes.ExternalTemplateImport(
+            template=otypes.Template(
+                name=module.params['name']
+            ),
+            url=external_template.get('url'),
+            cluster=otypes.Cluster(
+                name=module.params['cluster'],
+            ) if module.params['cluster'] else None,
+            storage_domain=otypes.StorageDomain(
+                name=external_template.get('storage_domain'),
+            ) if external_template.get('storage_domain') else None,
+            host=otypes.Host(
+                name=external_template.get('host'),
+            ) if external_template.get('host') else None,
+        )
+    )
+
+    # Wait until event with code 1158 for our template:
+    templates_service = connection.system_service().templates_service()
+    wait(
+        service=templates_service.template_service(imported_template.template.id),
+        condition=lambda tmp: len(events_service.list(
+            from_=int(last_event.id),
+            search='type=1158 and message=*%s*' % tmp.name,
```

Comment:
Currently, there is an issue in event search with templates the `message=*%s*` is a workaround for it. 
Opened an issue on it:
https://bugzilla.redhat.com/show_bug.cgi?id=1974656